lib/services/queue_service.dart

```diff
@@ -1162,6 +1181,7 @@ class QueueService {
         "android.media.extra.DOWNLOAD_STATUS": isDownloaded ? 2 : 0,
         "isOffline": FinampSettingsHelper.finampSettings.isOffline,
         "contextNormalizationGain": contextNormalizationGain,
+        "baseItem": item,
```

Comment:
there's already `itemJson` which can be used to get the `BaseItemDto` like so:
```dart
BaseItemDto.fromJson(item.extras!["itemJson"] as Map<String, dynamic>)
```

and each `FinampQueueItem` already includes this as a helper, as `FinampQueueItem.baseItem`.  

Storing the unserialized object in the extras probably doesn't work anyway.

Reply:
My bad! This was a leftover from earlier when I was still figuring out how to get an item out of a `FinampQueueItem` and ran down the wrong path (`queueItem.baseItem` was what I went with in the end).